.mailmap

```diff
@@ -0,0 +1,19 @@
+Cenk Gündoğan <mail-github@cgundogan.de> <cnkgndgn@gmail.com>
+Cenk Gündoğan <mail-github@cgundogan.de> <mail@cgundogan.de>
+Hauke Petersen <devel@haukepetersen.de> <hauke.petersen@fu-berlin.de>
+Hauke Petersen <devel@haukepetersen.de> <mail@haukepetersen.de>
+Joakim Nohlgård <joakim.nohlgard@eistec.se> <joakim.gebart@eistec.se>
+Joakim Nohlgård <joakim.nohlgard@eistec.se> <joakim@gebart.se>
+Kaspar Schleiser <kaspar@schleiser.de>
+Ludwig Knüpfer <ludwig.knuepfer@fu-berlin.de> <ludwig.ortmann@fu-berlin.de>
+Martine Lenders <m.lenders@fu-berlin.de> <authmillenon@gmail.com>
+Martine Lenders <m.lenders@fu-berlin.de> <mail@martin-lenders.de>
+Martine Lenders <m.lenders@fu-berlin.de> <mail@martine-lenders.eu>
+Martine Lenders <m.lenders@fu-berlin.de> <mlenders@inf.fu-berlin.de>
+Oleg Hahm <oleg@hobbykeller.org> <oleg@hobbykeller.org>
```

Comment:
@OlegHahm this line is necessary, otherwise it won't accumulate your other names that you used in conjunction with `oleg@hobbykeller`

BTW: are you:
```
2  oleg <oleg@scherbe.des-mesh.net>
```
?